### PR TITLE
Disable warnings for inline custom tags

### PR DESCRIPTION
**Pull request by SerheyDolgushev**

### CODE REVIEW

**Comment by andrerom** [comment]
Does this depend on any of the other PR's?

**Comment by SerheyDolgushev** [comment]
Nope

**Comment by andrerom** [comment]
Let me rephrase, the warning is still relevant on releases prior to 2.5.1 right?

**Comment by SerheyDolgushev** [comment]
If I`m not wrong inline custom tags were introduced in 2.5.0. So warning might be relevant on releases prior to 2.5.0.

**Comment by andrerom** [comment]
Maybe we can do a class exists check to see if we use Rich Text Bundle or not to know if we are on 1.13 or 2.5. WDYT @alongosz ?

**Comment by SerheyDolgushev** [comment]
In that case, this PR will also require Rich Text Bundle dependency. Maybe #91 and #93 can be released in a new major?

**Comment by andrerom** [comment]
Discussion on 2.5 bump or not here: https://github.com/ezsystems/ezplatform-xmltext-fieldtype/pull/92#discussion_r279266386
